examples/todomvc/components/itemview.go

```diff
@@ -52,6 +53,7 @@ func (p *ItemView) onStartEdit(event *vecty.Event) {
 	p.editing = true
 	p.editTitle = p.Item.Title
 	p.ReconcileBody()
+	p.input.Node().Call("focus")
```

Comment:
This is guaranteed to happen after `ItemView.render()` was called, right? Otherwise `p.input` will be `nil`.